com.unity.netcode.gameobjects/Runtime/Components/Interpolator/BufferedLinearInterpolator.cs

```diff
@@ -350,23 +428,63 @@ private void TryConsumeFromBuffer(double renderTime, float minDeltaTime, float m
         /// <param name="tickLatencyAsTime">The tick latency in relative local time.</param>
         /// <param name="minDeltaTime">The minimum time delta between the current and target value.</param>
         /// <param name="maxDeltaTime">The maximum time delta between the current and target value.</param>
+        /// <param name="isLerpAndExtrapolate">Determines whether to use smooth dampening or extrapolation.</param>
```

Comment:
nit: this param is a bit unclear
```suggestion
        /// <param name="isLerpAndExtrapolate">True when using <see cref="NetworkTransform.InterpolationTypes.Lerp"/>; otherwise false.</param>
```

Reply:
This is exactly as it should be.  👌 
`NetworkTransform.InterpolationTypes.Lerp` uses the `BufferedLinearInterpolator<T>.Update(float deltaTime, double renderTime, double serverTime)` method and `NetworkTransform.InterpolationTypes.LerpExtrapolateBlend` and ``NetworkTransform.InterpolationTypes.SmoothDampening` use the `BufferedLinearInterpolator<T>.Update(float deltaTime, double tickLatencyAsTime, double minDeltaTime, double maxDeltaTime, bool isLerpAndExtrapolate)` method when being updated by `NetworkTransform`.